packages/fluentui/react-bindings/src/icons/createSvgIcon.tsx

```diff
@@ -1,17 +1,40 @@
-import { getUnhandledProps, useStyles } from '@fluentui/react-bindings';
-import { Extendable } from '@fluentui/styles';
 import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { Extendable } from '@fluentui/styles';
+import * as customPropTypes from '@fluentui/react-proptypes';
 // @ts-ignore
 import { ThemeContext } from 'react-fela';
 
-import { ProviderContextPrepared } from '../../types';
-import { default as SvgIcon, SvgIconProps, SvgIconChildrenFn } from './SvgIcon';
+import { StylesContextValue } from '../styles/types';
+import useStyles from '../hooks/useStyles';
+import getUnhandledProps from '../utils/getUnhandledProps';
+import { SvgIconCreateFnParams, SvgIconProps } from './types';
+
+export const SvgIconClassName = 'ui-icon';
+export const SvgIconDisplayName = 'SvgIcon';
+
+export const SvgIconPropTypes = {
```

Comment:
We are going to remove `propTypes` at all. Should we drop them here?

Reply:
Make sense..